engine/schema/src/main/java/com/cloud/storage/SnapshotPolicyVO.java

```diff
@@ -59,14 +59,20 @@ public class SnapshotPolicyVO implements SnapshotPolicy {
     @Column(name = "uuid")
     String uuid;
 
+    @Column(name = "account_id")
+    long accountId;
+
+    @Column(name = "domain_id")
+    long domainId;
+
```

Comment:
query - do we sync them when a VM is moved between users?

Reply:
actually, I did not consider this case.